docs/markdown/Commands.md

```diff
@@ -423,8 +423,20 @@ format should be used. There are currently 3 formats supported:
 - `vscode`: Same as `sh` but without `$VAR` substitution because they do not
   seems to be properly supported by vscode.
 
-{{ devenv_arguments.inc }}
+*Since 1.7.0* `--bin <target>` argument can be used to build and run a given
+executable target. See [`compile`](#compile) command for `target` syntax.
```

Comment:
The name `--bin` is *extremely* nondescriptive for this. With this PR:

- `meson devenv gst-launch-1.0` will execute gst-launch-1.0
- `meson devenv --bin gst-launch-1.0` will run ninja to build gst-launch-1.0, and then launch gst-launch-1.0 itself

I strongly oppose this semantic interface.

Reply:
Removed --bin from devenv command.